presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -487,11 +488,15 @@ protected Node visitShowCreate(ShowCreate node, Void context)
                     sqlProperties.put(propertyName, sqlExpression);
                 }
 
+                List<Property> propertyNodes = sqlProperties.build().entrySet().stream()
```

Comment:
I think this can be named just `properties`

Reply:
There is already a local variable named `properties`